packages/jsii-rosetta/lib/commands/transliterate.ts

```diff
@@ -244,7 +200,7 @@ function transliterateType(
       );
```

Comment:
We need the `workingDirectory` here so that we can fixturize properly and find the snippet in `translateSnippet`, correct? Otherwise we would not find the snippet and fail with unknown snippet mode (I think).